File: game_server/ServerProtocol/src/Common/BagItemUseEnum/EBagItemUse_HeroType.java
package Common.BagItemUseEnum;

/*********
 * 背包使用道具-大臣类型
 **/
public enum EBagItemUse_HeroType {
	NONE, //0 ==== 
	POWER, //1 ==== 实力
	;
public static final EBagItemUse_HeroType[]  EBagItemUse_HeroType_Values = EBagItemUse_HeroType.values();
public static final int EBagItemUse_HeroType_Length = EBagItemUse_HeroType_Values.length;
public static EBagItemUse_HeroType EBagItemUse_HeroType_FromInt(int _ivalue) {
	if(_ivalue < 0 || _ivalue >= EBagItemUse_HeroType_Length){ return null; }
	return EBagItemUse_HeroType_Values[_ivalue];
}
}

File: game_server/ClientProtocol/Common/BagItemUseEnum/EBagItemUse_HeroType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ALBasicProtocolPack;
using UnityEngine;


namespace Common.BagItemUseEnum
{

/// <summary>
/// 背包使用道具-大臣类型
/// </summary>
public enum EBagItemUse_HeroType {
	NONE, //0 ==== 
	[InspectorName("POWER - idx[1] - 实力")]
	POWER, //1 ==== 实力
}

public class EBagItemUse_HeroTypeComparer : IEqualityComparer<EBagItemUse_HeroType>{
	public bool Equals(EBagItemUse_HeroType x, EBagItemUse_HeroType y) { return x == y; }
	public int GetHashCode(EBagItemUse_HeroType obj) { return obj.GetHashCode(); }
	public static int g_iEnumCount = 2;
}
}

